cms/envs/test.py

```diff
@@ -192,6 +192,10 @@
 # No segment key
 CMS_SEGMENT_KEY = None
 
+# For most tests, setting Jwt cookies during login is not necessary,
+# and raises errors when an oauth client has not been configured.
+FEATURES['ENABLE_LOGIN_SET_JWT_COOKIES'] = False
```

Comment:
I'm open to suggestions, but this is not a feature that is meant to be found and used generally.  Maybe it should be renamed to `DISABLE_LOGIN_SET_JWT_COOKIES_FOR_TESTS`, and then it may not matter as much?
If others feel strongly it should be in `*.envs.common`, then I think I would rename it.

Reply:
@jmbowman: I did rename this flag to make it more clear.  Given that, both @nasthagiri and I agreed that we wanted to keep it out of `common` because it isn't a flag that is intended to be used in any way other than how it is currently being used.